deployment/assets/create-script.py

```diff
@@ -2,15 +2,10 @@
 import json
 
 MODELS_LIST = json.loads(os.getenv('MODELS_LIST', ["mini"]))
-MODELS_DIR = json.loads(os.getenv('MODELS_DIR', ["sentence-transformers_all-MiniLM-L6-v2"]))
-TRANSFORMERS_CACHE_DIR = os.getenv('TRANSFORMERS_CACHE', "/opt/app-root/src/.cache/huggingface/hub").replace("hub", "")
 
 with open ('../../load-models-config.sh', 'w') as rsh:
     init_models_command = 'cd demo && '
     for model in MODELS_LIST:
-        init_models_command += f"cd models/{model} && cd ../../ &&"
-    init_models_command += ' cd server && '
-    for model_dir in MODELS_DIR:
-        init_models_command += f"cd {TRANSFORMERS_CACHE_DIR}sentence_transformers/{model_dir} && cd &&"
-    init_models_command += ' cd ' 
+        init_models_command += f"cd models/{model} && cd ../../../ &&"
```

Comment:
There is an extra ../ here.  If I use 2 models it cds into the first one but fails on the second like:
> cd: models/mini: No such file or directory

Because it went up a directory too far.
I think even with one model it fails with:

> ./load-models-config.sh: line 1: cd: server: No such file or directory

Reply:
Sorry, I need to document better; I included the model artifacts folder in there too, so the actual path that needs to be passed in `MODELS_LIST` is `["mini/sentence-transformer_mini.."]`. The full model folder path.

Reply:
so yeah, need to clarify this models structure.  I was actually thinking of the usual `models/<model-id>` structure but this is the "other" models folder w/ org/name.  Not all models have an org/ so at some point this would need to be clarified so anyone organizing an s3 model store understands the connection here.

Moving to caikit-nlp there is some interest in pulling models right from s3: urls w/o needing the volume.  Not sure when that might become available, but it sounds like it would be good for this.  Would probably add clarity if we just did artifacts_path = s3:... but I didn't expect us to add that